src/Altinn.Dan.Plugin.Statensvegvesen/Utils/Utils.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+
+namespace Altinn.Dan.Plugin.Statensvegvesen.Utils;
+
+public static class OedUtils
+{
+    public static string MapSsn(string ssn, string externalSource)
+    {
+        // todo Make this table driven and more maintainable when we have more interfaces and test users
+        // todo Differntiate between different test enviroments. Prod should allways return the input without modification.
+        if (externalSource == "svv")
+            switch (ssn)
+            {
+                case "12113701569":
```

Comment:
Could just remove the switch and keep only the svv values? Perhaps I should do the same for banking ... :|

Reply:
Agree, looking into it :-)